h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeSubgraph.java

```diff
@@ -131,9 +131,9 @@ void print() {
     rootNode.printEdges();
   }
 
-  void printDot(PrintStream os, int maxLevelsToPrintPerEdge, boolean detail, String optionalTitle, PrintMojo.PrintTreeOptions treeOptions) {
```

Comment:
I think the parameter name is too specific to the purpose we are using it for. I would call it "isDirected" or something like that.

Also, I would perhaps preserve the original signature of the function as well (in case MLI uses it).